src/plugins/explore/public/components/query_panel/components/footer/actions/index.tsx

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+// TODO: Actions are not scope of P0. This should bes tested with external plugins in next scope.
+
+import React, { useState } from 'react';
+import { i18n } from '@osd/i18n';
+import {
+  EuiButtonEmpty,
+  EuiContextMenuPanel,
+  EuiContextMenuItem,
+  EuiPopover,
+  EuiIcon,
+} from '@elastic/eui';
+import { queryBarActionsRegistry } from './registry';
```

Comment:
Import behaves differently in dev mode and prod mode, I guess the registry will be shared across multiple plugins and should be exposed through plugin's lifeCycle in a global singleton manner.

Related issue: https://github.com/opensearch-project/OpenSearch-Dashboards/issues/9899